Symfony/app/Resources/views/base.html.twig

```diff
@@ -42,10 +42,9 @@
                         <a href="#contact">Contact</a>
                     </li>
                     <li>
-                    {% if is_granted('ROLE_USER') %}
-                        <a href="./logout">Logout</a>
-                    {% endif %}
-                        {% if not is_granted('ROLE_USER') %}
+                        {% if is_granted('ROLE_USER') %}
```

Comment:
That assumes that *all* users will be granted such a role, which might or might not be true. For example an admin might or might not have that role but *still* be logged in and needing to find the link to log out.

In reality you're not looking for the role in this test, you're looking to see if authentication is performed (a user that is logged in). So maybe testing against for [`IS_AUTHENTICATED_REMEMBERED`](http://symfony.com/doc/2.3/cookbook/security/remember_me.html#forcing-the-user-to-re-authenticate-before-accessing-certain-resources) is a more global/safe check.